frontend/tailwind.config.js

```diff
@@ -1,9 +1,7 @@
 import colors from "tailwindcss/colors";
 import defaultTheme from "tailwindcss/defaultTheme";
 
-/**
- * @type {import("tailwindcss").Config}
- */
+/** @type {import("tailwindcss").Config} */
```

Comment:
Do we need `@ts-check` here too like postcss.js

Reply:
@natsukagami it sorta helps improve UX so yeah (IDEs will report if we pass an invalid configuration).